recruiter/views/authentication.py

```
@@ -1,122 +1,46 @@
 
 from django.http.response import HttpResponse, HttpResponseRedirect
-from django.shortcuts import render, get_object_or_404
+from django.shortcuts import render
 from django.conf import settings
 from django.urls import reverse
 from django.template import loader, Template, Context
-from django.template.loader import render_to_string
-from django.core.exceptions import ObjectDoesNotExist
 from datetime import datetime
 from django.utils import timezone
+from zoneinfo import ZoneInfo
```

Ruff (0.11.9)

`zoneinfo.ZoneInfo` imported but unused

Remove unused import: `zoneinfo.ZoneInfo`

(F401)